src/i18n/en-system-messages.ts

```diff
@@ -7,6 +7,7 @@ const systemMessagesMessageDescriptors = {
       ${SystemMessagesName.EnoughExitRecoveryApprovals} {Enough permission holders have now signed to exit recovery mode. As long as no further storage slots are updated, a recovery permission holder may now sign a transaction to reactivate the colony.}
       ${SystemMessagesName.MotionHasPassed} {{motionTag} has {passedTag} and may be finalized.}
       ${SystemMessagesName.MotionRevealPhase} {It's time to {revealTag} votes to the world!}
+      ${SystemMessagesName.MotionFullyStaked} {{motionTag} is fully staked; Staking period has reset. As long as there is no {objectionTag}, the motion will pass.}
```

Comment:
I noticed that in Figma there is little space between `objectionTag` and a coma that you don't have and I think it looks a bit better with little space. But maybe not too important.